api/proto/core/monitor/alert/alert.proto

```diff
@@ -903,15 +903,19 @@ message AlertNotifySilence {
 }
 
 message GetAlertRequest {
-  int64 id = 1 [(validate.rules).int64.gt = 0];
```

Comment:
You'd better keep `id` as field number 1, and append `scope` & `scopeId` as 2,3, to ensure forward compatibility.
See: https://protobuf.dev/programming-guides/proto3/#assigning-field-numbers